src/components/UI/DefaultInput.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { TextInput, StyleSheet } from 'react-native';
+
+import styleVariables from '../../style-variables';
+
+const defaultInput = props => (
+  <TextInput
+    {...props}
+    style={[
+      styles.input,
+      props.style,
+      !props.valid && props.touched ? styles.invalid : null
+    ]}
+    underlineColorAndroid='transparent'
+  />
+)
+
+const styles = StyleSheet.create({
```

Comment:
This should go in a separate styles.js file.

Reply:
👍